unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarVisibility.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+namespace DCL
+{
+    public class AvatarVisibility : MonoBehaviour
```

Comment:
We added this new behaviour for a number of reasons:
1. Other avatars behave differently than the own avatar. We wanted to handle that difference in one unique place
1. Hiding other avatar requieres to hide other game objects. For example the canvas & the passport collider
1. Previously, avatars were hidden by deactivating the avatar gameobject. However, we need when this happened, the position wasn't modified and the collider stopped triggering events. So we hide the new complexity in this behavour
1. Changing camera mode, would switch the own avatar on and off. Now, if we are in an area that hides the avatar, this needs to change. So we add the ability to "lock" whether the avatar is displayed or not